src/config.rs

```diff
@@ -2,44 +2,49 @@ use crate::net::constants::{DEFAULT_MTU, FRAGMENT_SIZE_DEFAULT, MAX_FRAGMENTS_DE
 use std::{default::Default, time::Duration};
 
 #[derive(Clone)]
-/// Struct that contains config values for various aspects of the network
+/// Contains the configuration options to configure laminar for special use-cases.
 pub struct Config {
-    /// The maximal amount of time to keep `VirtualConnection`s around before cleaning them up.
+    /// Value which can specify the amount of time to keep the connection alive before cleaning them up.
     pub idle_connection_timeout: Duration,
-    /// This is the maximal size a packet can get with all its fragments.
+    /// Value which can specify the maximal size a packet can get with all its fragments.
     ///
     /// Recommended value: 16384
     pub max_packet_size: usize,
-    /// These are the maximal fragments a packet could be divided into.
+    /// Value which can specify the maximal allowed fragments.
     ///
     /// Why can't I have more than 255 (u8)?
     /// This is because you don't want to send more then 256 fragments over UDP, with high amounts of fragments the chance for an invalid packet is very high.
     /// Use TCP instead (later we will probably support larger ranges but every fragment packet then needs to be resent if it doesn't get an acknowledgement).
     ///
-    /// Recommended value: 16 but keep in mind that lower is better.
+    /// default: 16 but keep in mind that lower is better.
```

Comment:
The maximum number of fragments into which a packet can be split. Defaults to 16. Keep in mind lower is better.